src/Mentat.UI/Models/Assignment.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace Mentat.UI.Models
+{
+    public class Assignment
```

Comment:
Properties should be pascal-case

Reply:
@jonnywwright I got that updated, the rest of the PRs are outdated as I change the PR to development

Reply:
I see now. I'll resolve those other comments and approve.